scripts/parser/backend/test_cmakecachevariable.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from cmakecachevariable import cmakecache_variable as cmc_var
```

Comment:
Even if we're talking all the time about cmakecache variables, putting this "cache" suffix _everywhere_ creates a lot of clutter. Isn't better to just use cmake_var? This will save us from using cmc_var etc.

Reply:
Done.